### DISCUSSION TITLE
Customize notification of showNotification

### DISCUSSION THREAD

**Discussion by Meierschlumpf** [mantinedev/mantine#3373] [Feature requests]
Hey

Can I add an optional parameter to NotificationProvider to allow custom Notifications with no property changes. I would love to see that, because I want to add translations to my notifications and I'm unable to use a hook with the showNotification method.

Is that okay for you, or should I work arround for myself? 

```tsx
const showServerErrorNotification = () => {
  showErrorNotification({
    title: t("common:error.server.title"),
    message: t("common:error.server.message"),
  });
};
```

**Comment by rtivital** [comment]
I do not understand the issue. Why you cannot use `showNotification` function?

**Reply by Meierschlumpf** [reply]
I wanted to add translations to the notifications with i18n, but I just noticed, that I can use jsx within the title and message property and use the Trans component of i18n. So the issue is already fixed. My bad.

```tsx
export const showServerErrorNotification = () => {
  showErrorNotification({
    title: <Trans i18nKey="common:error.server.title" />,
    message: <Trans i18nKey="common:error.server.message" />,
  });
};
```
